src/remarkable-ui/components/editors/selects/MultiSelectField/MultiSelectField.stories.tsx

```
@@ -0,0 +1,249 @@
+  },
+};
+
+export const WithCategories: Story = {
+  args: {
+    label: undefined,
+    required: false,
+    options: mockOptionsWithCategories,
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)